src/styles/less/Hud.less

```diff
@@ -277,10 +277,12 @@
 
 //State not done for this PT21
 .currentDay {
-  display: flex;
-  flex-flow: row wrap;
-  background-color: var(--green);
-  animation: pulse-animation 2s infinite;
+  color: var(--green);
+  text-shadow: -1px -1px 0 rgb(117, 117, 117), 1px -1px 0 rgb(117, 117, 117),
```

Comment:
I like the idea of the text shadow, but it might make it slightly harder to read as it removes some of the contrast between the green text and gray background; however, this could just be me, but it's something to consider.